ui/src/lib/yjs/useYNode/utils.ts

```diff
@@ -0,0 +1,235 @@
+import type { Node } from "@flow/types";
+
+import {
+  reassembleEdge,
+  yEdgeConstructor,
+  yNodeConstructor,
+} from "../conversions";
+import type { YEdge, YEdgesArray, YNodesArray, YWorkflow } from "../types";
+
+export function updateParentYWorkflowNode(
+  currentWorkflowId: string,
+  yParentWorkflow: YWorkflow,
+  node: Node,
+  params: any,
+) {
+  const yParentNodes = yParentWorkflow?.get("nodes") as YNodesArray | undefined;
```

Comment:
Remove the ? from yParentWorkflow?.get("nodes"). The function expects yParentWorkflow to always be defined. The type assertion is correct though, since get("nodes") might return undefined.